### ISSUE TITLE
Tagging inside Code blocks

### ISSUE THREAD

**Issue by hasantahir** [glushchenko/fsnotes#1399]
**Describe your feature request**
Currently, the character `#` generates a tag even if it is inside a code block. For example,
```
#include<stdio.h>
```
generates a `include<stdio.h>` tag. One way to overcome this is to use `# include`. Is it possible to disable tagging within a code block?

**Additional context**
Tested on FS Notes 6 on macOS.

**Comment by hasantahir** [comment]
@glushchenko. Saw your message upon the release of Beta 5. I believe this should also have bene labelled as a Bug but I am unable to change it now.

Thanks.

**Comment by glushchenko** [comment]
Can not reproduce it, please provide example.

**Comment by hasantahir** [comment]
A MWE would be:

```
#include<stdio.h>


which is being displayed as:

<img width="243" alt="image" src="https://user-images.githubusercontent.com/11236148/198563051-f1c8fb68-360d-426f-a587-ef8b1f8e21e9.png">


and rendered as:

<img width="589" alt="image" src="https://user-images.githubusercontent.com/11236148/198563110-a5d2f8f7-2051-4f3f-a09f-4484d0569ddc.png">

**Comment by glushchenko** [comment]
<img width="807" alt="image" src="https://user-images.githubusercontent.com/1089917/199051525-465e152d-8a37-48a6-bdf7-c6420f49fdd8.png">

**Comment by hasantahir** [comment]
Thanks for getting back. This is what happens on my end:

Tried this on the published version 6.0.0 (543). Any suggestions on where to look further?


https://user-images.githubusercontent.com/11236148/199054574-1563aaf5-7568-431a-9645-dad97ba57e6d.mov

**Comment by glushchenko** [comment]
Pfff, i dont know why this dont recognised like code block, may be non unicode chars or etc. Just use ⌘ + control + c shortcut. Works fine for me.
